db/migrate/20100716055919_convert_messages.rb

```diff
@@ -1,24 +1,30 @@
 class ConvertMessages < ActiveRecord::Migration
   def self.up
-    if const_defined?('ThinkingSphinx')
+    if const_defined?("ThinkingSphinx")
       ThinkingSphinx.deltas_enabled = false
     end
 
-    partners = Message.find_by_sql("SELECT DISTINCT sender_id, recipient_id FROM messages")
-    partners = partners.map{|m| [m.sender_id, m.recipient_id].sort}.uniq
-    partners = partners.map{|p| p.map{|i| User.find(i)}}
+    partners = Message.find_by_sql(
+      "SELECT DISTINCT sender_id, recipient_id FROM messages"
+    )
+    partners = partners.map { |m| [m.sender_id, m.recipient_id].sort }.uniq
+    partners = partners.map { |p| p.map { |i| User.find(i) } }
 
     partners.each do |user1, user2|
-      puts "#{user1.username} -> #{user2.username}"
       messages = Message.find(
         :all,
-        conditions: ['(sender_id = ? AND recipient_id = ?) OR (sender_id = ? AND recipient_id = ?)', user1.id, user2.id, user2.id, user1.id],
-        order:      'created_at ASC',
-        include:    [:sender, :recipient]
+        conditions: [
+          "(sender_id = ? AND recipient_id = ?) OR " +
+            "(sender_id = ? AND recipient_id = ?)",
+          user1.id, user2.id, user2.id, user1.id
+        ],
+        order: "created_at ASC",
+        include: [:sender, :recipient]
       )
 
       threaded_messages = {}
-      subject           = "Conversation between "+[user1.username, user2.username].sort.join(' and ')
+      subject = "Conversation between " +
+                [user1.username, user2.username].sort.join(" and ")
```

Comment:
Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.